webapp/src/blocks/fileBlock.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Block, createBlock} from './block'
+import {ContentBlock} from './contentBlock'
+
+type FileBlockFields = {
+    attachmentId: string
+}
+
+type FileBlock = ContentBlock & {
```

Comment:
Should be named "Attachment" or "AttachmentBlock", following names for other blocks. "Card" / "CommentBlock". Essentially, name should match "type". 

Also, should not be a "ContentBlock".

Reply:
Okay